docs/tutorial.md

```diff
@@ -0,0 +1,656 @@
+## Installing django-readers
+
+Install from [PyPI](https://pypi.org/project/django-readers/):
+
+```shell
+pip install django-readers
+```
+
+## Prerequisites
+
+This tutorial assumes you understand the basics of Django (models, views, templates etc). If you don't, make sure you've been through [the Django tutorial](https://docs.djangoproject.com/) first. The section on Django REST framework assumes that you've read and understood [the docs](https://www.django-rest-framework.org/).
+
+## Example models
+
+For the purposes of this tutorial, we're going to use a set of models representing **books**, **authors** and **publishers**. A book has exactly one publisher, but can have many authors. Here are the Django models we're using:
+
+
+```python
+class Publisher(models.Model):
+    name = models.CharField(max_length=100)
+    address = models.TextField()
+
+
+class Author(models.Model):
+    name = models.CharField(max_length=100)
+    email = models.EmailField()
+
+
+class Book(models.Model):
+    title = models.CharField(max_length=100)
+    authors = models.ManyToManyField(Author)
+    publisher = models.ForeignKey(Publisher)
+    publication_date = models.DateField()
+```
+
+We're going to go through a step-by-step set of requirements from an imaginary client who has asked us to build a simple book listing application. We're going to assume the Django project itself has already been set up, and focus on a single view and a single template.
+
+## Requirement 1: a list of book titles (without `django-readers`)
+
+First, the client wants us to build a page containing a list of all of the books in the database, showing only their titles. The standard Django approach to this would be as follows:
```

Comment:
```suggestion
First, we want to build a page containing a list of all of the books in the database, showing only their titles. The standard Django approach to this would be as follows:
```

I'm thinking not everyone reading these docs will be working with clients so might read a bit strangely to them.

If you agree with this change, don't forget to change it in other places too.

Reply:
i actually prefer using the pronoun "you" as I am closer to the Google styleguide as mentioned here in https://ddbeck.com/first-person-plural-is-questionable/

Reply:
Agreed about "you" vs "we", I'll tweak this. I think using an imaginary client as the motivation for each step in the tutorial is probably OK, a lot of people working with Django do have clients, and those that don't can substitute a product manager (or whoever) making the requests.